File: app/src/main/java/com/example/fibonacciapp/Location.java
package com.example.fibonacciapp;

import android.widget.ImageView;

import com.google.gson.annotations.SerializedName;

public class Location {

    private int id;

    private String title;

    private String name;
    @SerializedName("pictures")
    private ImageView image;

    public int getId() {
        return id;
    }

    public String getTitle() {
        return title;
    }

    public String getName() {
        return name;
    }

    public ImageView getImage() {
        return image;
    }
}

File: app/src/main/java/com/example/fibonacciapp/db/enitity/Location.kt
package com.example.fibonacciapp.db.enitity

import androidx.room.*

import com.google.gson.annotations.SerializedName

const val LOCATION_TABLE_ID = 1

@Entity
data class Location (

    var id: Int = 0,
    @ColumnInfo(name = "location_title")
    var title: String? = null,
    @ColumnInfo(name = "location_name")
    var name: String? = null,
    @ColumnInfo(name = "location_image")
    @TypeConverters(StringListConverter::class)
    var pictures: MutableList<String>?
)  {
    @PrimaryKey(autoGenerate = false)
   var idLocation: Int = LOCATION_TABLE_ID
}
